docs/getting-started.md

```diff
@@ -1,21 +1,22 @@
 # Getting Started
 
-| Features| CE | PRO
-|-------|-----| ------
-|- [Install](#install)| ✅ | ✅
-|- [Create an API](#create-an-api)| ✅ | ✅
-|- [Access an API](#access-an-api)| ✅ | ✅
-|- [Secure an API](#secure-an-api)| ✅ | ✅
-|- [Publish an API](#publish-an-api-to-the-tyk-portal)| ❌ | ✅
+| Features                                            | CE  | PRO | Description                                                            |
+|-----------------------------------------------------|-----|-----|------------------------------------------------------------------------|
+| [Install](#install)                                 | ✅   | ✅   | -                                                                      |
+| [Create an API](#create-an-api "hover")             | ✅   | ✅   | -                                                                      |
+| [Access an API](#access-an-api)                     | ✅   | ✅   | -                                                                      |
+| [Secure an API](#secure-an-api)                     | ❌   | ✅   | `Secure an API` is not yet supported for CE.                           |
```

Comment:
It is definitely possible to secure an API using Tyk CE. 

Security Policy API was non existent for Tyk CE, however I believe it was released with R4.
You don't need a security policy to secure an API.

Reply:
Thanks @asoorm for the clarification. So, Tyk Operator is capable of managing security policies for CE and PRO installations, right?

Reply:
Security Policies need to be implemented.

https://github.com/TykTechnologies/tyk-operator/blob/master/pkg/client/gateway/security_policy.go

---

The point I was making is that you can still secure an api without security policies. So I would change to a warning: "Security Policies not yet implemented for Tyk CE, and a link to appropriate issue". https://github.com/TykTechnologies/tyk-operator/issues/357